.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,37 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: Before opening a new issue, please search for duplicate or closed issues.
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [improvement, mistake, other]
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Include as much detail as possible, for example:
+        - A (perma)link to the documentation in question
+        - Suggestions for how to change the docs
+    validations:
+      required: true
+
+  - type: checkboxes
+    attributes:
+      label: Contributing
+      description: >-
+        It takes a lot of work to maintain `simple-icons`, so we welcome any
+        contributions!
+      options:
+        - label: I'm willing to submit a PR for this
```

Comment:
This [will be rendered as a checkbox in the issue](https://github.com/sachinraja/simple-icons/issues/1) as well, I wonder if it makes sense to change it into a drowdown so it just has the either the text "I don't plan to contribute this icon" or "I plan to contribute this icon" (wording open to discussion) (former being the default).

Also not sure about the description, I would prefer something more _boring_, e.g. "This is an open source project and we welcome contributions, do you want to contribute this icon?"

Reply:
> This will be rendered as a checkbox in the issue as well

That's unfortunate, a dropdown does make more sense in that case.

> "This is an open source project and we welcome contributions, do you want to contribute this icon?"

Sounds good to me, though something more general would be nice so they are the same across all the forms.

Reply:
> That's unfortunate, a dropdown does make more sense in that case.

:+1: 

> Sounds good to me, though something more general would be nice so they are the same across all the forms.

At least for me they don't all need the exact same wording, as long as the "template" is the same it's fine. But a generic text that works for all template is just as good :+1: